src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppPlugin.scala

```diff
@@ -89,10 +99,14 @@ object SbtReactiveAppPlugin extends AutoPlugin {
 
         localPath.setExecutable(true)
 
+        target
+      },
+
+      dockerCommands := {
         val addCommand = startScriptLocation
           .value
           .toVector
-          .map(path => docker.Cmd("ADD", localPath.getAbsolutePath, path))
+          .map(path => docker.Cmd("COPY", localName, path))
```

Comment:
Does this mean the `rp-start` script is expected to end up in the same directory as the working directory of the docker build?

Reply:
On checking the code, I think this is going to happen due to `rp-start` being present in `src/main/resources`.

Reply:
Yes, the `docker:stage` task writes it to the working directory for the docker build.